core/tests/performance_tests/base.py

```diff
@@ -36,9 +40,22 @@ def _record_page_metrics_from_cached_session(self, page_url):
             self.data_fetcher.get_page_metrics_from_cached_session(page_url))
 
     def _record_page_timings_for_url(self, page_url):
```

Comment:
Yes, ``_record_average...`` sounds good to me and I think also resolved @maitbayev's comment.